pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -579,6 +579,11 @@ type TiFlashSpec struct {
 	// +optional
 	Config *TiFlashConfigWraper `json:"config,omitempty"`
 
+	// Initializer is the configurations of the init container for TiFlash
+	//
+	// +optional
+	Initializer *InitContainerSpec `json:"initializer,omitempty"`
```

Comment:
```suggestion
	InitContainer *InitContainerSpec `json:"initContainer,omitempty"`
```

Reply:
`initContainers` is a field in `ComponentSpec`,so I think `initContainer` a little confusing

Reply:
OK